TTS/config/shared_configs.py

```diff
@@ -210,6 +210,13 @@ class BaseTrainingConfig(Coqpit):
         tb_model_param_stats (bool):
             Enable / Disable logging internal model stats for model diagnostic. It might be useful for model debugging.
             Defaults to ```False```.
+            wandb_disabled: bool = False
+        wandb_project_name (str):
```

Comment:
We can take the name from `run_name`above

Reply:
The project name is the name of W&B project that is created. One project can contain many runs. For naming the runs, `run_name` from above is used. The project name, if not provided will default to `config.model`. If it makes more sense to just remove this arg and force the project name to be `config.model`, we can do that too.

Reply:
Thanks for the explanation. I think it is fine to keep it as it is then.